manifests/c/chrisant996/Clink/1.2.8.516b24/chrisant996.Clink.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: chrisant996.Clink
+PackageVersion: 1.2.8.516b24
+PackageLocale: en-US
+Publisher: Christopher Antos
+PublisherUrl: https://github.com/chrisant996/
+PublisherSupportUrl: https://github.com/chrisant996/clink/issues
+#PrivacyUrl: 
+Author: Chris Antos
+PackageName: Clink
+PackageUrl: https://chrisant996.github.io/clink/
+License: GNU General Public License v3.0
+LicenseUrl: https://raw.githubusercontent.com/chrisant996/clink/master/LICENSE
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: Clink combines the native Windows shell cmd.exe with the powerful command line editing features of the GNU Readline library, which provides rich completion, history, and line-editing capabilities.
+Description: Clink combines the native Windows shell cmd.exe with the powerful command line editing features of the GNU Readline library, which provides rich completion, history, and line-editing capabilities. Readline is best known for its use in the well-known Unix shell Bash, the standard shell for Mac OS X and many Linux distributions.
+Moniker: clink
+Tags:
+- command line
```

> Note: The best practice is to present these terms in all lower case with hyphens rather than spaces.

From: [schema/1.0.0/defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
- command-line
- command
```